app/views/users/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="user index">
+    <h2>All Registered Users</h2>
+    <table class="listing" summary="User List">
+        <tr class="header">
+            <th>#</th>
+            <th>Name</th>
+            <th>Email</th>
+            <th>Role</th>
+            <th>Actions</th>
+        </tr>
+        <%@users.each do |user| %>
+        <tr>
+            <td><%= user.id%></td>
+            <td><%= user.name%></td>
+            <td><%= user.email%></td>
+            <td><%= user.role%></td>
+            <td class="actions">
+                 <%=link_to("Show", user_path(user),:class => "action edit")%>
+                <%=link_to("Edit", edit_user_path(user),:class => "action edit")%>
+                <%# <%=link_to("Delete",delete_user_path(user),:class => "action delete")%>
+            </td>
+        </tr>
+        <%end%>
```

Comment:
if no user is there may be we can show something like no user found rather than keeping the table body empty
Thanks!

Reply:
Modified this bhai